### PR TITLE
Remove broken ChargeType link

### PR DESCRIPTION
**Pull request by flanakin**
Removing a link that was accidentally left in 0.5 when we moved the allowed values into the column spec.

### CODE REVIEW

**Review by salpaysenturus — APPROVED** [review]
Awesome, looks good!  I'll admit I didn't even see the values had been embedded in the column definition because I couldn't get past the broken link.  That might sound dumb, but I'm sure other people have done the same!

File: specification/dimensions/chargetype.md [hunk 1]
```diff
@@ -25,17 +23,17 @@ after-the-fact adjustment (e.g., credits), or taxes.
...
+| Data type       | String         |
+| Allows nulls    | False          |
+| Value format    | list-of-values |
 
 Allowed values:
 
 | Value      | Description                                                                                                                                                                   |
-|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| :--------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Adjustment | Any adjustments that are applied after the original usage or purchase record. Adjustments may be related to multiple charges.                                                 |
```
Comment (SanjnaSrivatsaVMware) [inline specification/dimensions/chargetype.md:37]: Do out-of-cycle bills, refunds and credits apply as an Adjustment?

Comment (flanakin) [reply specification/dimensions/chargetype.md:37]: Refunds and credits on those out-of-cycle bills would be adjustments. Other line items would show as they normally would. We're more focused on the cost details than invoice generation today.

Comment (flanakin) [reply specification/dimensions/chargetype.md:37]: Let's discuss in Slack if you think there's a gap here we should account for in 1.0.

**Review by ljadvey — APPROVED** [review]

**Review by udam-f2 — APPROVED** [review]

**Pull request merged**